### PR TITLE
Updated the fetch plugins scripts to pull from our repo

### PR DESCRIPTION
**Pull request by cman131**
I also removed the commented out commands

### CODE REVIEW

**Comment by CShapleigh** [comment]
So how much does the output change when you remove the debug/verbose flag? Because some of the output could be useful

**Comment by cman131** [comment]
@CShapleigh It is much more concise (mostly hides the git output)
It seemed sufficient that it lists out the plugins at the end. I can put the "-d"s back in if we want

**Comment by thenaterhood** [comment]
I realize this isn't the point of this pull, but at some point this could be switched to use Python's subprocess library. Also, I'm not sure why there's separate scripts for windows other than 'sudo'. Could just require it to be run with sudo on Linux.

**Comment by cman131** [comment]
Yeah, I wanted to do that. Actually I would prefer to change the files to bash and bat scripts. Would we be cool with that?

**Comment by thenaterhood** [comment]
(We spoke offline)

It looks like the two Python scripts are identical other than the fact that the Linux one runs everything with sudo. We can probably just drop that one and use the "windows" one then update the instructions to run the script as root, rather than maintaining two scripts.

**Comment by CShapleigh** [comment]
I like that idea @thenaterhood

File: fetchPlugins.py [hunk 1]
```diff
@@ -1,32 +1,28 @@
 '''
 This script fetches all the required plugins automatically.
 Assumes cordova has been added successfully to the project.
-Requires sudo access. Mac/Linux support only
```
Comment (thenaterhood) [inline fetchPlugins.py]: It may be a good idea to say that on Mac/Linux it requires sudo access still.

Comment (cman131) [inline fetchPlugins.py]: does it actually require it though?

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Team Centri-Pedal Motion (RIT 2014) ran into an issue where there were a few Cor
 	antlib.xml:584: shellscript returned: 1`
 
 ###Required Cordova Plugins
-Below is a list of the required plugins. We have included a script that will fetch these automatically by simply running `python fetchPlugins.py` for mac, or `python fetchPluginsWindows.py` for Windows. This assumes that the locations of these repos are still as written below.
+Below is a list of the required plugins. We have included a script that will fetch these automatically by simply running `python fetchPlugins.py`. This assumes that the locations of these repos are still as written below.
```
Comment (bbesmanoff) [inline README.md]: Since the scripts now reflect the TourTrak repos, you may want to update the links below

Comment (thenaterhood) [inline README.md]: Good call @bbesmanoff! :+1:

Comment (cman131) [inline README.md]: :+1: completed

File: fetchPlugins.py (hunk 1 above)
Comment (cman131) [inline fetchPlugins.py]: "sudo" decidedly unnecessary from slack discussion

**Comment by bbesmanoff** [comment]
:jack_o_lantern:

**Comment by thenaterhood** [comment]
:jack_o_lantern:

**Pull request merged**